spec/requests/api/v1/users/sign_in_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'User login', type: :request do
+  before do
+    post api_v1_user_session_path, params: user_params, as: :json
+  end
+
+  context 'when user credentials are correct' do
+    let(:password) { Faker::Internet.password }
+    let(:user) { User.create!(attributes_for(:user, password: password)) }
+    let(:user_params) { { email: user.email, password: password } }
+
+    it 'returns ok status with authorization token' do
+      expect(response).to have_http_status(:created)
+      expect(response.body).to eq({ message: 'Logged in successfully.' }.to_json)
+      # binding.pry
+      expect(response.headers).to have_key('client')
+      expect(response.headers).to have_key('access-token')
```

Comment:
We should try to have as little amount of expectations per example, trying to always aim for only 1 expectation per example